Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG PY_VERSION=3.7
+FROM continuumio/miniconda3:4.8.2-alpine AS builder
+
+EXPOSE 8888
+
+LABEL maintainer.name="mosdef-hub"\
+      maintainer.url="https://mosdef.org"
+
+ENV PATH /opt/conda/bin:$PATH
+
+USER root
+
+ADD . /mbuild
+
+WORKDIR /mbuild
+
+RUN conda update conda -yq && \
+	conda config --set always_yes yes --set changeps1 no && \
+	conda config --add channels omnia && \
+	conda config --add channels conda-forge && \
+	conda config --add channels mosdef && \
+	conda create -n mbuild-docker python=$PY_VERSION && \
+	. /opt/conda/etc/profile.d/conda.sh && \
+	conda activate mbuild-docker && \
+	conda install python=$PY_VERSION nomkl --file requirements-dev.txt && \
+        python setup.py install && \
+	echo "conda activate mbuild-docker" >> \
+	/home/anaconda/.profile && \
+	conda clean -afy && \
+	chown -R anaconda:anaconda /mbuild && \
+	chown -R anaconda:anaconda /opt && \
+	chown -R anaconda:anaconda /home/anaconda
+
+USER anaconda
+
+WORKDIR /scratch
```

Comment:
It might be nicer to change the WORKDIR if the user is `anaconda`.

Reply:
yeah I currently have no real preference.

im not sure what the best practice is in this case. I like the general scratch area, and most likely the user will be mounting some files/directories themselves, so maybe this will not be used much. I can be swayed either way.

Reply:
Yeah. This is the entrypoint directory for the image. It could be defaulted to `/home/anaconda` IMO.